plugins/inputs/nvidia_smi/schema_v12/types.go

```diff
@@ -0,0 +1,257 @@
+		} `xml:"fbc_stats"`
+		GpuFruPartNumber string `xml:"gpu_fru_part_number"`
+		GpuModuleID      string `xml:"gpu_module_id"`
+		GpuOperationMode struct {
+			CurrentGom string `xml:"current_gom"`
+			PendingGom string `xml:"pending_gom"`
+		} `xml:"gpu_operation_mode"`
+		GpuPartNumber    string `xml:"gpu_part_number"`
+		GpuPowerReadings struct {
+			CurrentPowerLimit   string `xml:"current_power_limit"`
+			DefaultPowerLimit   string `xml:"default_power_limit"`
+			MaxPowerLimit       string `xml:"max_power_limit"`
+			MinPowerLimit       string `xml:"min_power_limit"`
+			PowerDraw           string `xml:"power_draw"`
+			PowerState          string `xml:"power_state"`
+			RequestedPowerLimit string `xml:"requested_power_limit"`
+		} `xml:"gpu_power_readings"`
+		GpuResetStatus struct {
+			DrainAndResetRecommended string `xml:"drain_and_reset_recommended"`
+			ResetRequired            string `xml:"reset_required"`
+		} `xml:"gpu_reset_status"`
+		GpuVirtualizationMode struct {
+			HostVgpuMode       string `xml:"host_vgpu_mode"`
+			VirtualizationMode string `xml:"virtualization_mode"`
+		} `xml:"gpu_virtualization_mode"`
+		GspFirmwareVersion string `xml:"gsp_firmware_version"`
+		Ibmnpu             struct {
+			RelaxedOrderingMode string `xml:"relaxed_ordering_mode"`
+		} `xml:"ibmnpu"`
+		InforomVersion struct {
+			EccObject  string  `xml:"ecc_object"`
+			ImgVersion string  `xml:"img_version"`
+			OemObject  float64 `xml:"oem_object"`
+			PwrObject  string  `xml:"pwr_object"`
+		} `xml:"inforom_version"`
+		MaxClocks struct {
+			GraphicsClock string `xml:"graphics_clock"`
+			MemClock      string `xml:"mem_clock"`
+			SmClock       string `xml:"sm_clock"`
+			VideoClock    string `xml:"video_clock"`
+		} `xml:"max_clocks"`
+		MaxCustomerBoostClocks struct {
+			GraphicsClock string `xml:"graphics_clock"`
+		} `xml:"max_customer_boost_clocks"`
+		MigDevices string `xml:"mig_devices"`
+		MigMode    struct {
+			CurrentMig string `xml:"current_mig"`
+			PendingMig string `xml:"pending_mig"`
+		} `xml:"mig_mode"`
+		MinorNumber         string `xml:"minor_number"`
+		ModulePowerReadings struct {
+			CurrentPowerLimit   string `xml:"current_power_limit"`
+			DefaultPowerLimit   string `xml:"default_power_limit"`
+			MaxPowerLimit       string `xml:"max_power_limit"`
+			MinPowerLimit       string `xml:"min_power_limit"`
+			PowerDraw           string `xml:"power_draw"`
+			PowerState          string `xml:"power_state"`
+			RequestedPowerLimit string `xml:"requested_power_limit"`
+		} `xml:"module_power_readings"`
+		MultigpuBoard string `xml:"multigpu_board"`
+		Pci           struct {
+			AtomicCapsInbound  string `xml:"atomic_caps_inbound"`
+			AtomicCapsOutbound string `xml:"atomic_caps_outbound"`
+			PciBridgeChip      struct {
+				BridgeChipFw   string `xml:"bridge_chip_fw"`
+				BridgeChipType string `xml:"bridge_chip_type"`
+			} `xml:"pci_bridge_chip"`
+			PciBus         int64  `xml:"pci_bus"`
```

Comment:
This is not an int ;) it will need to be a string as it should be a hex value. On my system it is showing up as "0A", here is my output: https://pastebin.com/VGCB4mQF

I'm not sure about the others like PCIDevice and PCIDomain